controllers/reconcilers/kserve_istio_servicemonitor_reconciler.go

```diff
@@ -70,6 +70,16 @@ func (r *KserveIstioServiceMonitorReconciler) Reconcile(ctx context.Context, log
 	return nil
 }
 
+func (r *KserveIstioServiceMonitorReconciler) Delete(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
```

Comment:
Once this class implement `SingleResourcePerNamespace`, its not required to provide empty definition of delete funtion here

Reply:
Thanks, I missed that :) Fixed in 28944bc